frontend/src/pages/Landing/index.js

```diff
@@ -228,7 +228,13 @@ function Landing({ user }) {
   const [alertReportsCount, setAlertReportsCount] = useState(0);
   const [filters, setFilters] = useState([]);
   const [alertFilters, setAlertFilters] = useState([]);
-  const [appliedRegion, updateAppliedRegion] = useState(user.homeRegionId === 14 ? 14 : regions[0]);
+
+  const defaultRegion = regions[0] || user.homeRegionId || 0;
```

Comment:
Do we need the third option here (setting it to 0)?

Reply:
User's do not necessarily have a `homeRegionId`. the zero catches the case where the user isn't assigned any regions and does not have a `homeRegionId`